source/context.puml

```diff
@@ -1,35 +1,43 @@
 @startuml S-ENDA context diagram
-!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Container.puml
+!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Context.puml
 
 LAYOUT_LEFT_RIGHT
 
-Person(researcher, "Researcher")
-Person(developer, "Developer")
-Person(scientistproducer, "Scientist data producer")
+Boundary(producers, "Producers") {
```

Comment:
I think is should be "providers" instead of "producers", shouldn't it? Looks more in agreement with the blue boxes and the text..